<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="rupesh" failures="1" tests="1" name="Add_Report_Test" time="62.306" errors="0" timestamp="26 Mar 2020 08:46:02 GMT">
  <testcase classname="com.dataseers.qa.testcases.AddReport_Test" name="Add_NewReport_Test" time="58.651">
    <failure type="java.lang.AssertionError" message="Save new report scenario failed">
      <![CDATA[java.lang.AssertionError: Save new report scenario failed
	at com.dataseers.qa.pages.Administration_Module.ds_Report_Mgmt_Page.savereport(ds_Report_Mgmt_Page.java:162)
	at com.dataseers.qa.testcases.AddReport_Test.Add_NewReport_Test(AddReport_Test.java:59)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Add_NewReport_Test -->
</testsuite> <!-- Add_Report_Test -->
